plugins/feeds/public/malpedia_malware.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from datetime import timedelta
+from typing import ClassVar
+
+from core import taskmanager
+from core.schemas import entity, task
+
+
+class Malpedia_Malware(task.FeedTask):
+    _defaults = {
+        "frequency": timedelta(days=1),
+        "name": "Malpedia",
+        "description": "Gets list of malpedia malwares",
+        "source": "https://malpedia.caad.fkie.fraunhofer.de/",
+    }
+
+    _SOURCE: ClassVar["str"] = (
+        "https://malpedia.caad.fkie.fraunhofer.de/api/get/families"
+    )
+
+    def run(self):
+        response = self._make_request(self._SOURCE)
+        if not response:
+            return
+        families_json = response.json()
+        for sign_mal, entry in families_json.items():
+            self.analyze_entry_malware(sign_mal, entry)
+
+    def analyze_entry_malware(self, name_malware, entry: dict):
+        """Analyzes an entry as specified in the malpedia json."""
+        try:
+            m = entity.Malware(
+                name=entry["common_name"], description=entry["description"]
+            )
+        except Exception as e:
+            logging.error(f"Error creating malware entity: {e}")
+            return
```

Comment:
Why catch the error here but noot in the previous one? What error are you expecting?

Reply:
Same comment as above - you're not loading anything from the database here.

Reply:
some entries has no name in malpedia

Reply:
In that case let's check the name and skip the creation attempt instead of try / catching